src/main/java/ProjectOcean/Controller/ApplicationController.java

```diff
@@ -75,14 +75,18 @@ public void showStudyPlanWorkspaceWindow() {
     @FXML
     private void onDragOver(DragEvent event) {
         Movable draggedObject = (Movable) event.getGestureSource();
-        moveDraggedObjectToCursor(draggedObject, event);
+        relocateDraggedObjectToCursor(draggedObject, event);
         event.consume();
     }
 
     @FXML
     private void onDragDone(DragEvent event) {
         Movable draggedObject = (Movable) event.getGestureSource();
         getChildren().remove(draggedObject);
```

Comment:
Just a thought: which container is the 'children' from? Can it be specified, like contentWindow.getChildren()?

Reply:
from applicationController. Added this. before to make it clearer.